frontend/src/components/NavBar/PrintImage/form.tsx

```diff
@@ -0,0 +1,650 @@
+  classes,
+  handleClose,
+  setTitleText,
+  debounceCallback,
+  country,
+  mapDimensions,
+  setMapDimensions,
+  logo,
+  logoPosition,
+  setLogoPosition,
+  logoScale,
+  setLogoScale,
+  toggles,
+  setToggles,
+  legendPosition,
+  setLegendPosition,
+  setFooterText,
+  footerTextSize,
+  setFooterTextSize,
+  download,
+  defaultFooterText,
+  selectedBoundaries,
+  setSelectedBoundaries,
+  legendScale,
+  setLegendScale,
+  handleDownloadMenuOpen,
+  handleDownloadMenuClose,
+  downloadMenuAnchorEl,
+}: DownloadFormUIProps) {
+  const { t } = useSafeTranslation();
+
+  // list of expandible section
+  const [expandedSections, setExpandedSections] = React.useState<
+    ExpandedSections
+  >({
+    logos: false,
+    labels: false,
+    adminAreas: false,
+    legend: false,
+    footer: false,
+  });
+
+  const toggleExpanded = useCallback(
+    (section: keyof typeof expandedSections) => () => {
+      setExpandedSections(prev => ({ ...prev, [section]: !prev[section] }));
+    },
+    [expandedSections],
+  );
+
+  return (
+    <div className={classes.optionsContainer}>
+      <div>
+        <Box fontSize={14} fontWeight={900} mb={1} className={classes.title}>
+          {t('Map Options')}
+        </Box>
+        <IconButton
+          className={classes.closeButton}
+          onClick={() => handleClose()}
+        >
+          <Cancel />
+        </IconButton>
+      </div>
+
+      {/* Title */}
+      <div className={classes.optionWrap}>
+        <TextField
+          defaultValue={country}
+          placeholder={t('Title')}
+          fullWidth
+          size="small"
+          inputProps={{ label: t('Title'), style: { color: 'black' } }}
+          onChange={event => {
+            debounceCallback(setTitleText, event.target.value);
+          }}
+          variant="outlined"
+        />
+      </div>
+
+      {/* Width */}
+      <ToggleSelector
+        value={mapDimensions.width}
+        options={mapWidthSelectorOptions}
+        setValue={val =>
+          setMapDimensions((prev: MapDimensions) => ({
+            ...(prev || {}),
+            width: val as number,
+          }))
+        }
+        title={t('Map Width')}
+      />
+
+      {/* Logo */}
+      {logo && (
+        <SectionToggle
+          title={t('Logo')}
+          classes={classes}
+          expanded={toggles.logoVisibility}
+          handleChange={({ target }) => {
+            setToggles(prev => ({
+              ...prev,
+              logoVisibility: Boolean(target.checked),
+            }));
+            setLogoPosition(target.checked ? 0 : -1);
+          }}
+        >
+          <GreyContainer>
+            <GreyContainerSection>
+              <Box display="flex" flexDirection="row">
+                <ToggleSelector
+                  value={logoPosition}
+                  options={logoPositionOptions}
+                  iconProp={logoPosition}
+                  setValue={setLogoPosition}
+                  title={t('Logo Position')}
```

Comment:
Since there are now in sections you can probably get rid of the "Logo" prefix here and other prefixes elsewhere.
Note that we might need to adjust the translation files